app/utils/RequestCountry.scala

```diff
@@ -1,13 +1,12 @@
 package utils
 
 import com.gu.i18n.CountryGroup
-import com.netaporter.uri.Uri
-import com.netaporter.uri.dsl._
 import play.api.mvc.Request
 
 object RequestCountry {
   implicit class RequestWithFastlyCountry(r: Request[_]) {
-    def getFastlyCountry = r.headers.get("X-GU-GeoIP-Country-Code").flatMap(CountryGroup.byFastlyCountryCode)
+    def getFastlyCountryGroup = r.headers.get("X-GU-GeoIP-Country-Code").flatMap(CountryGroup.byFastlyCountryCode)
+    def getFastlyCountry = r.headers.get("X-GU-GeoIP-Country-Code").flatMap(CountryGroup.countryByCode)
```

Comment:
Agreed, there is a bad name in membership-common. The cost/benefit ratio of fixing that is quite large as it'll require releasing all 3 apps which are dependent on membership-common. So I've raised a ticket in Trello: https://trello.com/c/MZwyk9HK/89-fix-countrygroup-byfastlycountrycode-api-nomenclature-in-membership-common